docs/data/toolpad/studio/components/list/List.js

```diff
@@ -1,6 +1,8 @@
+/* eslint-disable filenames/match-exported */
 import * as React from 'react';
 import { List } from '@toolpad/studio-components';
 
+// TODO fix name
```

Comment:
What's wrong with the name?

Reply:
filenames/match-exported rule wants the component to be called List but List is already in the scope.

I left the TODO to keep this a small time investment, my main objective was for new code to be less likely to bend the shared eslint rules. For sure, we can fix it in this PR too.

Reply:
I updated the naming everywhere.